features/polyExplorer/src/screens/stories/digitalGiantsStory.jsx

```diff
@@ -45,6 +46,99 @@ const DigitalGiantsStory = () => {
         ...new Set(jurisdictionLinks.map(({ target }) => target)),
     ].filter((j) => j !== "EU-GDPR");
 
+    const listOfDataCategories = Object.keys(
+        globalData.personal_data_categories
+    );
+
+    let totalShares = 0;
+    bigSix.forEach((company) => {
+        company._data.dataTypesShared.forEach((i) => (totalShares += i.count));
+    });
+
+    const dataTypesSharedCombined = listOfDataCategories
+        .map((category) => {
+            let total = 0;
+            bigSix.map((company) => {
+                company._data.dataTypesShared.forEach((typeCategory) => {
+                    if (typeCategory["dpv:Category"] === category)
+                        total += typeCategory.count;
+                });
+            });
+            return total !== 0
+                ? {
+                      "dpv:Category": category,
+                      total,
+                  }
+                : null;
+        })
+        .filter((e) => e)
+        .sort((a, b) => b.total - a.total);
+
+    const [selectedDataTypeBubble, setSelectedDataTypeBubble] = useState(
+        dataTypesSharedCombined[0].total
+    );
+
+    const dataTypes = [
+        {
+            id: "by-companies",
+            label: "By Companies",
```

Comment:
This is not translated?

Reply:
It is translated (please see the translation above). I assume we needed a "label" without the translation for the Tabs because we weren't using the "id". However, since we are using the "id" now, I'm not sure if I can use the translation in the "label" and delete the "translation", add the translation to both "label" and "translation" or keep it as it is right now. @RichardSto could you please clarify? 😄